start_jupyter.sh

```diff
@@ -59,7 +59,6 @@ fi
 
 # Run Python code using the selected interpreter with all environment variables
 export PYTHONPATH=$PYTHONPATH
-source "$(dirname "$(readlink -f "$0")")/first_bash_script_env.sh"
```

Comment:
Why this is deleted?

Reply:
It's an outdated file which does no longer exist in this repo